sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobClient.java

```diff
@@ -164,25 +165,58 @@ public void upload(InputStream data, long length, boolean overwrite) {
      * @param requestConditions {@link BlobRequestConditions}
      * @param timeout An optional timeout value beyond which a {@link RuntimeException} will be raised.
      * @param context Additional context that is passed through the Http pipeline during the service call.
+     * @deprecated
```

Comment:
Do we need to also deprecate in EncryptedBlobClient?

Reply:
I think it would also be helpful to link the method to use instead of this method as part of the deprecation message.